src/twisted/web/http.py

```diff
@@ -1224,6 +1227,13 @@ def write(self, data):
 
             for name, values in self.responseHeaders.getAllRawHeaders():
                 for value in values:
+                    if name == b"Content-Length" and isOkay and isCONNECT:
+                        self._log.info(
```

Comment:
as commented by Tom,

I think is best to have this for a separate PR/ticket and raise a deprecation warning rather than a log warning.

and in the future raise an exception.

Or maybe just raise an exception right away as this can be considered a bug :)